tests/plugins/test_import_image.py

```diff
@@ -256,25 +256,10 @@ def test_ensure_primary(tmpdir, monkeypatch, osbs_error, tag_conf, annotations,
      .with_args(TEST_IMAGESTREAM)
      .and_return(ImageStreamResponse()))
 
-    # By using a combination of ordered and once, we verify that
-    # ensure_image_stream_tag is not called with version-release tag
-    for x in range(DEFAULT_TAGS_AMOUNT):
-        expectation = (
-            flexmock(OSBS)
-            .should_receive('ensure_image_stream_tag')
-            .with_args(dict, tag_prefix + str(x))
-            .once()
-            .ordered()
-        )
-        if osbs_error:
-            expectation.and_raise(OsbsResponseException('None', 500))
+    repository = '{0}/{1}'.format(TEST_REGISTRY, TEST_REPO)
```

Comment:
can be shorten to
    repository = '{}/{}'.format(TEST_REGISTRY, TEST_REPO)

Reply:
done